crates/flox/src/utils/metrics.rs

```diff
@@ -200,8 +200,7 @@ async fn push_metrics(metrics: Vec<MetricEntry>, uuid: Uuid) -> Result<()> {
         })
         .collect::<Result<Vec<serde_json::Value>>>()?;
 
-    let client = reqwest::Client::new();
-    let res = client
+    reqwest::Client::new()
```

Comment:
clippy?

Reply:
`res` was unused, and I dropped `client` because it wasn't needed